src/config/config.js

```diff
@@ -6,8 +6,7 @@ module.exports = {
     password: process.env.DB_PASSWORD,
     database: process.env.DB_NAME,
     host: process.env.DB_HOST,
-    dialect: 'postgres',
-    operatorsAliases: false
```

Comment:
why did you remove `operatorsAliases`?

Reply:
operatorsAliases has been deprecated and shows as a warning in the console.

Reply:
ok